### PR TITLE
Add test coverage reporting

### PR DESCRIPTION
**Pull request by alecgibson**
Example coverage report:
code coverage for content performance manager

### CODE REVIEW

**Comment by alecgibson** [comment]
[Add support for test coverage reporting](https://trello.com/c/h2hTpiP5/367-add-support-for-test-coverage-reporting)

**Comment by andrewgarner** [comment]
Do we want to generate a coverage report for every test run or only in certain circumstances like a CI build? Some other apps have made coverage optional – https://github.com/alphagov/asset-manager/commit/235ca57053742d72d1bada6f68708270e47ff1b8

File: Gemfile [hunk 1]
```diff
@@ -52,4 +52,5 @@ group :development, :test do
   gem 'spring-watcher-listen'
   gem 'timecop'
   gem 'webmock'
+  gem 'simplecov', :require => false
```
Comment (alecgibson) [inline Gemfile]: @andrewgarner done.

**Comment by alecgibson** [comment]
@andrewgarner @pmanrubia When do we want to be running this? What's it going to be used for? (ie do we want the build to fail if we don't have sufficient test coverage?)

**Comment by andrewgarner** [comment]
The [documentation suggests the overhead should be minimal](https://github.com/colszowka/simplecov#running-coverage-only-on-demand) – maybe we could measure this?

**Comment by alecgibson** [comment]
@andrewgarner I have to say that I anecdotally didn't notice a difference compared to our current plethora of integration tests. I'd say it'll take ages to recoup the time we waste trying to measure it.

**Comment by alecgibson** [comment]
As discussed with @pmanrubia we won't fail the build on test coverage.

**Pull request merged**